### PR TITLE
Feature/perf bar

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: templates/partials/hero.html [hunk 1]
```diff
@@ -14,5 +14,8 @@ <h1 id="hero-heading" class="text--display">Explore Campaign Finance Data</h1>
           </div>
       </form>
     </div>
+  <script type="application/javascript">
```
Comment (jmcarp) [inline templates/partials/hero.html]: Should this be wrapped in `{% if debug %}...{% endif %}`?

File: templates/layouts/main.html [hunk 2]
```diff
@@ -15,9 +15,23 @@
...
     {% if api_location %}
     <script>
-        API_LOCATION = "{{api_location}}"
-        API_VERSION = "{{api_version}}"
-        API_KEY = "{{api_key}}"
+        API_LOCATION = "{{api_location}}";
+        API_VERSION = "{{api_version}}";
+        API_KEY = "{{api_key}}";
+
+        function trackMetric(name, imagesSelector) {
```
Comment (jmcarp) [inline templates/layouts/main.html]: Ditto below: Only for development mode, right?

File: templates/partials/hero.html (hunk 1 above)
Comment (msecret) [inline templates/partials/hero.html]: I actually want these performance marks to be present all the time. I can explain more in a meeting, but essentially these are custom metrics that we could track for various pages. If we keep these marks, it means that performance tracking tools like new relic, or google analytics will automatically pick them up.

File: templates/layouts/main.html (hunk 2 above)
Comment (msecret) [inline templates/layouts/main.html]: Same as above

File: templates/partials/hero.html (hunk 1 above)
Comment (jmcarp) [inline templates/partials/hero.html]: Oh, that's cool. Didn't know `window.performance.mark` was a thing!

File: __init__.py [hunk 3]
```diff
@@ -54,6 +54,7 @@ def _get_default_cycles():
 app.jinja_env.globals['api_location'] = config.api_location
 app.jinja_env.globals['api_version'] = config.api_version
 app.jinja_env.globals['api_key'] = config.api_key_public
+app.jinja_env.globals['debug'] = config.debug
```
Comment (jmcarp) [inline __init__.py]: Is this used in the jinja templates?

Comment (msecret) [inline __init__.py]: No, would it be useful at all?

Comment (jmcarp) [inline __init__.py]: Doesn't seem like we have a use for it right now. Might as well take it out of the jinja globals until we need it.

**Comment by jmcarp** [comment]
@msecret do we have to do anything to get the extra performance metrics into New Relic?

**Pull request merged**